content/6_Advanced/LCT.mdx

```diff
@@ -215,7 +233,22 @@ int main() {
 }
 ```
 
-### With Euler-Tour Tree
+### With Euler Tour Tree
+
+An alternative solution is to use the Euler Tour Tree (ETT) technique.
```

Comment:
i believe the usaco guide has a module on the euler tour, can you link that?

Reply:
Should I add it as a resource or just a normal hyperlink?

Reply:
just a normal hyperlink
but i'm dumb so can you check if this euler tour is the same as the one covered in the guide? if it isn't then add an explanation on how this differs from the other euler tour

Reply:
Done